ibm_catalog.json

```diff
@@ -0,0 +1,66 @@
+{
+  "products": [
+    {
+      "name": "terraform-ibm-secrets-manager-iam-engine",
+      "label": "IAM secrets engine module",
+      "product_kind": "module",
+      "tags": [
+        "dev_ops",
+        "target_terraform",
+        "terraform",
+        "module"
+      ],
+      "keywords": [
+        "terraform",
+        "iam",
+        "secrets",
+        "encryption"
```

Comment:
I would remove `iam`, `secrets` and `encryption` and replace with `iam-credentials-engine` and `secrets-manager`

Reply:
@ocofaigh AFAIK, the keywords are used for searching. For example, `secrets` pulls up `Secrets Manager` in the catalog. Wouldn't we use it here (or are you thinking we'd use that keyword for a the SM module)? 

Also, terms with dashes don't seem like search keywords. "iam engine", "iam credentials engine", "secrets manager" would seem appropriate.